programs/survival/revive_coral

```diff
@@ -0,0 +1,16 @@
+//allows player to revive dead corals with water bottles
+__on_player_right_clicks_block (player, item_tuple,hand, block,face,hitvec) -> (
+	l(item, count, nbt) = item_tuple || l('None', 0, null);
+	dead=l('dead_brain_coral_block','dead_bubble_coral_block','dead_fire_coral_block','dead_horn_coral_block','dead_tube_coral_block');
+	live=l('brain_coral_block','bubble_coral_block','fire_coral_block','horn_coral_block','tube_coral_block');
+	listpos=dead ~ block;
+	if(item=='potion'&& dead ~ block !=null,
+		print('WOrks');
```

Comment:
Now, we know it WOrks, you can remove it :)

Reply:
o crap I forgot thanks